### PR TITLE
feat: remove user from MLS conversation (#FS-561)

### PR DESCRIPTION
**Pull request by PatrykBuniX**
BREAKING CHANGES: 

- renamed `removeUser` function to `removeUserFromProteusConversation` in `@wireapp/core`
- changed the types of conversation and user id in `@wireapp/bot-api` `MessageHandler` from `string` to `QualifiedId` (new api requirement)

## Pull Request Checklist

- [x] My code is covered by tests
- [x] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/core/src/main/conversation/ConversationService/ConversationService.ts [hunk 1]
```diff
@@ -1260,4 +1263,49 @@ export class ConversationService {
       conversation,
     };
   }
+
+  private async sendCommitBundleRemovalMessages(groupIdDecodedFromBase64: Uint8Array, commitBundle?: CommitBundle) {
+    const coreCryptoClient = this.coreCryptoClientProvider();
+
+    if (commitBundle?.welcome) {
+      await this.apiClient.api.conversation.postMlsWelcomeMessage(Uint8Array.from(commitBundle.welcome));
+    }
```
Comment (thisisamir98) [inline packages/core/src/main/conversation/ConversationService/ConversationService.ts:1272]: Aren’t these welcome & commit already of type uint8array? why do we need to convert?

Comment (PatrykBuniX) [reply packages/core/src/main/conversation/ConversationService/ConversationService.ts:1272]: yeah, they are actually!

Comment (PatrykBuniX) [reply packages/core/src/main/conversation/ConversationService/ConversationService.ts:1270]: but it doesn't work for some reason, it throws an error: not enough bytes
Screenshot 2022-08-24 at 16 40 38

Comment (PatrykBuniX) [reply packages/core/src/main/conversation/ConversationService/ConversationService.ts:1272]: ^that one appears on adding user to mls and this one below on removing:
Screenshot 2022-08-24 at 18 05 27

File: packages/bot-api/src/MessageHandler.ts [hunk 2]
```diff
@@ -78,9 +78,9 @@ export abstract class MessageHandler {
     return this.account!.service!.user.getUsers(userIds);
   }
 
-  async removeUser(conversationId: string, userId: string): Promise<void> {
+  async removeUser(conversationId: QualifiedId, userId: QualifiedId): Promise<void> {
```
Comment (PatrykBuniX) [inline packages/bot-api/src/MessageHandler.ts:81]: I'm not sure where this function is being used and whether it's not a breaking change 🤔

Comment (aweiss-dev) [reply packages/bot-api/src/MessageHandler.ts:81]: expect it to be a breaking change

Comment (PatrykBuniX) [reply packages/bot-api/src/MessageHandler.ts:81]: //will release as breaking change

**Review by aweiss-dev — APPROVED** [review]

**Review by thisisamir98 — APPROVED** [review]

**Pull request merged**